src/l500/l500-device.h

```diff
@@ -79,6 +79,10 @@ namespace librealsense
         void update_flash_internal(std::shared_ptr<hw_monitor> hwm, const std::vector<uint8_t>& image, std::vector<uint8_t>& flash_backup,
             update_progress_callback_ptr callback, int update_mode);
 
+        ivcam2::extended_temperatures get_temperatures();
+        void start_temperatures_fetcher();
```

Comment:
Should these be private?

Reply:
It is called from ac-trigger & l500_temperature_options::query() so it needs to be public